tiledb/sm/crypto/crypto.h

```diff
@@ -158,6 +158,14 @@ class Crypto {
    */
   static Status sha256(
       const void* input, uint64_t input_read_size, Buffer* output);
+
+  /**
+   * Generates a number of cryptographically random bytes.
```

Comment:
A single line of documentation for something acting as a PRNG is inadequate.
* How's the IV initialized?
* Where's the state held between calls?
* Is the state global?
* Is it thread safe?

As long as these functions were not exposed to the outside, these issues didn't matter as much. If you're going to run tests against them, it matter. And the one test that does anything would pass with poor RNGs such as CRC shift or linear congruential.

Better yet. Simplify this PR and eliminate this function and its do-almost-nothing tests.

Reply:
Also. We have a PRNG in the code that has all this documented. We don't need another, and we certainly do not want another that's poorly documented.